crates/holochain/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ default_semver_increment_mode: !pre_minor beta-dev
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- The app validation workflow had been retriggered only when the number of concurrent ops to be validated (50) was reached. In many scenarios that doesn't happen and thus the workflow wasn't retriggered, even though some ops might not have been validated. Now the workflow will return `WorkComplete::Incomplete` when not all ops could be validated, which retriggers the workflow.
```

Comment:
```suggestion
- Fixes problem where in many cases app validation would not be retriggered for ops with missing dependencies. Previously the app validation workflow had been retriggered only when the number of concurrent ops to be validated (50) was reached. Now the workflow will be retriggered whenever any ops could not be validated.
```

Reply:
It's not only missing dependencies, but any op for which validation failed.